test/CadlRanchProjects/README.md

```diff
@@ -0,0 +1,13 @@
+# How to add a cadl ranch test
+
+1. Checkout this PR by `gh pr checkout 2563`
+2. Run `npm install` at the root folder
+3. Get your cadl ranch test name under `node_modules\@azure-tools\cadl-ranch-specs\http\*\<test name>`, e.g., `primitive-properties`
+4. Add this test name to `$cadlRanchProjectNames` in `eng/Generate.ps1`  https://github.com/Azure/autorest.csharp/blob/f185a6678597096d4a984a3c3664261688f0f50d/eng/Generate.ps1#L241
```

Comment:
Curious: why does this need to be manually added to the script?

Reply:
We will replace the configuration.json with sidecar. So we don't want to take the effort on that.